test/bidir-full-sync-tests.js

```diff
@@ -47,6 +48,107 @@ test2('bidirectional full sync test', getClusterSizes(5), 20000,
 	})
 );
 
+test2('bidirectional full sync throttling test', getClusterSizes(3), 20000,
+    prepareCluster(function(t, tc, n) {
+        return [
+            dsl.assertStats(t, tc, n+1, 0, 0),
+            dsl.waitForEmptyPing(t, tc),
+
+            assertReverseFullSyncThrottling(t, tc, 0, n*3, 5),
+
+            dsl.assertStats(t, tc, n+1, 0, 0),
+        ];
+    })
+);
+
+// assert if the number of joins is throttled.
+function assertReverseFullSyncThrottling(t, tc, idx, count, expectedJoins) {
+    var functions = [];
```

Comment:
I prefer if all the functions are just declared within the [] brackets, since I think it improves readability and is more in line with the rest of the integration tests.